apps/services/models.py

```diff
@@ -124,17 +147,28 @@ class UserRequestReview(Base):
     total_sum = Column(DECIMAL(7, 2))
     payment_deadline_date = Column(DATE)
     remark = Column(VARCHAR(length=255))
-    bed_place_id = Column(INTEGER, ForeignKey("bed_place.bed_place_id"))
+    bed_place_id = Column(
+        INTEGER,
+        ForeignKey("bed_place.bed_place_id", ondelete="CASCADE", onupdate="CASCADE"),
```

Comment:
ondelete value isn't correct, i think it should be do nothing or set null...

Reply:
Ok, I'd copied it from main.sql. There was a pack of issues, may be this also. It would be a good idea to give me a strict task - nothing or set null. ?